src/components/SeatLayout/SeatLayout.tsx

```diff
@@ -46,46 +49,29 @@ const SeatLayout = ({
                             {row.map((seat: number, index: number) => {
                                 if (seat === 1) {
                                     const currentSeat = seats[seatIndex];
+                                    const seatStatus = getSeatStatus(
+                                        currentSeat,
+                                        selectedSeats
+                                    );
+                                    console.log(seat,seatStatus)
```

Comment:
Please remove console.log

Reply:
Resolved in 8f43a3ef1367e03c27a0fcbc051361e0396c5c2c